golang/README.md

```diff
@@ -50,39 +61,67 @@ Instructions for setting up manual testing can be found in the README in
 
 ## API
 
-Cosmos modules expose messages, which modify state, and queries, which read
-state.
-
-### Queries
-
-Queries are available via CLI or REST. For more information, see the
-descriptions in the CLI.
-
-| Query | Description |
-| ----- | ----------- |
-| IsAncestor | Deteremine if a block is an ancestor of another |
-| GetRelayGenesis | Get the trusted root of the relay |
-| GetLastReorgLCA | Get the LCA of the latest reorg |
-| GetLastReorgLCA | Get the best digest known to the relay |
-| FindAncestor | Find the nth ancestor of a block|
-| IsMostRecentCommonAncestor | Determine if a block is the LCA of two headers|
-| HeaviestFromAncestor | Check which of two descendents is heaviest from the LCA |
-| GetRequest | Get details of an SPV Proof Request|
-| CheckProof | Check the syntactic validity of an SPV Proof |
-| CheckRequests | Perform CheckProof and check the SPV Proof against a set of Requests |
-
-### Messages
-
-Messages are available via CLI or REST. For more information, see the
-descriptions in the CLI.
-
-| Message | Description |
-| ------- | ----------- |
-| IngestHeaderChain | Add a chain of headers to the relay |
-| IngestDifficultyChange | Add a chain of headers to the relay with a difficulty change|
-| MarkNewHeaviest | Mark a new best-known chain tip |
-| NewRequest | Register a new SPV Proof request |
-| ProvideProof | Provide a proof that satisfies 1 or more requests |
+Cosmos modules expose queries (which read state) and messages (which modify
+state).  These are available via CLI or REST.
+
+### CLI
```

Comment:
Should probably make a note both here and in rest, that our convention is to use bitcoin hashes in their LE format. So a digest will end in 0s, not begin with 0s. `0xabcd...0000`, not `0x0000...cdab`

Reply:
Ok, added :)